qiskit/circuit/library/standard_gates/xx_minus_yy.py

```diff
@@ -0,0 +1,171 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Two-qubit XX-YY gate."""
+
+from typing import Optional
+
+import numpy as np
+
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.library.standard_gates.ry import RYGate
+from qiskit.circuit.library.standard_gates.rz import RZGate
+from qiskit.circuit.library.standard_gates.s import SdgGate, SGate
+from qiskit.circuit.library.standard_gates.sx import SXdgGate, SXGate
+from qiskit.circuit.library.standard_gates.x import CXGate
+from qiskit.circuit.parameterexpression import ParameterValueType
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+from qiskit.circuit.quantumregister import QuantumRegister
+from qiskit.qasm import pi
+
+
+class XXMinusYYGate(Gate):
+    r"""XX-YY interaction gate.
+
+    A 2-qubit parameterized XX-YY interaction. Its action is to induce
+    a coherent rotation by some angle between :math:`|00\rangle` and :math:`|11\rangle`.
+
+    **Circuit Symbol:**
+
+    .. parsed-literal::
+
+             ┌───────────────┐
+        q_0: ┤0              ├
+             │  {XX-YY}(θ,β) │
+        q_1: ┤1              ├
+             └───────────────┘
+
+    **Matrix Representation:**
+
+    .. math::
+
+        \newcommand{\th}{\frac{\theta}{2}}
+
+        R_{XX-YY}(\theta, \beta)\ q_0, q_1 =
```

Comment:
Is that `\` here on purpose? 🙂 (Same comment for the equation below with switched endianness)
```suggestion
        R_{XX-YY}(\theta, \beta) q_0, q_1 =
```